editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -1214,6 +1214,8 @@ export const NavigatorRowLabel = React.memo((props: NavigatorRowLabelProps) => {
               ? 'component'
               : props.emphasis === 'subdued'
               ? 'subdued'
+              : props.emphasis === 'emphasized'
+              ? 'primary'
```

Comment:
isn't primary a blue? I think that's a little too strong for emphasized

Reply:
yes it's blue, but that's also shared with other elements in the sample components so I wanted to keep it consistent 🤔

Reply:
ok let's merge and we can tweak it!